content/sensu-go/5.16/api/entities.md

```diff
@@ -28,7 +28,10 @@ The `/entities` API endpoint provides HTTP GET access to [entity][1] data.
 The following example demonstrates a request to the `/entities` API endpoint, resulting in a JSON array that contains the [entity definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/entities -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/entities \
+"Authorization: Bearer $SENSU_TOKEN" \
```

Comment:
should there be a `-H` at the start of this line?

Reply:
Yes, thank you